browser/src/layer/tile/CommentListSection.ts

```diff
@@ -1120,6 +1137,11 @@ export class CommentSection extends CanvasSectionObject {
 				}
 			}
 		} else if (action === 'Modify') {
+			if (app.view.commentAutoSave) {
+				app.view.commentAutoSave.sectionProperties.autoSave.innerText = _('Autosaved');
+				app.view.commentAutoSave = null;
```

Comment:
please add `commentAutoSave` property to the app.view declaration and comment what it does

Reply:
similar with `commentAutoAdded`